### ISSUE TITLE
gcis:Project, Model as property

### ISSUE THREAD

**Issue by justgo129** [USGCRP/gcis-ontology#96]
Like https://github.com/USGCRP/gcis-ontology/issues/95 this is a spinoff of #12.  Here, the turtle for models, in addition to projects, incorrectly uses classes as properties:
https://github.com/USGCRP/gcis/blob/master/lib/Tuba/files/templates/model/object.ttl.tut

Example with instance data:
http://data.globalchange.gov/model/ccsm3.thtml

I was unable to locate a candidate "property" relating a model to a Project like CMIP5 within dbpedia.

Similarly with relating a model run to a model:
https://github.com/USGCRP/gcis/blob/master/lib/Tuba/files/templates/model_run/object.ttl.tut

**Comment by zednis** [comment]
To verify, we are looking for a property to relate reference a project instance from a model instance.

What are the semantics here?  Did the project create the model, use the model, something different?

**Comment by justgo129** [comment]
As an example, we're trying to relate ccsm3 (model) with CMIP3 (project), and vice-versa.  I'd say the model is part of the project (okay, the running of the model is part of the project).  I'm not sure dcterms:isPartOf or dcterms:hasPart would be adequate.

**Comment by xgmachina** [comment]
FYI, `gcis:Project` is a subclass of `prov:Agent`. 
Should I say a model can be used in many projects through model runs.

**Comment by zednis** [comment]
I do not think either `dcterms:isPartOf` or `dcterms:hasPart` are appropriate.

Also, our conceptual modeling here will be dependent on the resolution of #17.

If we resolve #17 with the addition of the class `gcis:ModelRunOutput` (subclass of `prov:Entity`) then we can use PROV to say that the ModelRunOutput was attributed to the Project (note: I am assuming we are wanting to say that the project created the model run output, not that they project has used the model run output to do something else)

**Comment by justgo129** [comment]
Sounds good.  I'll defer to the judgement of @rewolfe and @bduggan on this one.

**Comment by justgo129** [comment]
Seeing no opinion to the contrary, please go ahead and perform the steps described in your latest comment, @zednis.

**Comment by justgo129** [comment]
Also @zednis just as a point of information, this may also allow us to resolve #12, as the only other instances of that issue are visible at:

https://github.com/USGCRP/gcis/blob/master/lib/Tuba/files/templates/model/object.ttl.tut
https://github.com/USGCRP/gcis/blob/master/lib/Tuba/files/templates/model_run/object.ttl.tut
https://github.com/USGCRP/gcis/blob/master/lib/Tuba/files/templates/project/object.ttl.tut

Basically, the material relevant to the discussions of the relationships between a model, model_run, and project.

**Comment by justgo129** [comment]
Now that #17 is resolved (thanks, @xgmachina), let's make the changes suggest by @zednis.  We can then close this issue and #12.

**Comment by justgo129** [comment]
@zednis when you get a chance please inform whether we close this issue.  Thanks!

**Comment by zednis** [comment]
I believe this ticket is closeable.

**Comment by justgo129** [comment]
Thanks, @zednis.  Closed #96 given merged https://github.com/USGCRP/gcis/pull/235.
